### DISCUSSION TITLE
How to test Desktop page with user flows?

### DISCUSSION THREAD

**Discussion by ipadorusa** [GoogleChrome/lighthouse#13572] [Q&A] [answered]
i entered th pc url but it redirects to mobile URL.

TT. 
How to PC URL TEST?

`const URL = 'PC URL'
const browser = await puppeteer.launch({
    headless: true,
    ignoreHTTPSErrors: true,
    args: [`--window-size=1920,1080`],
    defaultViewport: {
      width:1920,
      height:1080
    }
  });
  const page = await browser.newPage();
  const flow = await startFlow(page, {name: 'Single Navigation'});
  await flow.navigate(URL);`

**Comment by adamraine** [comment, marked as answer]
Doing `flow.navigate` will use the default Lighthouse viewport settings (Moto G4) regardless of the viewport you set with puppeteer beforehand. You can use the desktop viewport by using the Lighthouse desktop config:

```js
const config = require('lighthouse/lighthouse-core/config/desktop-config.js');
const flow = await startFlow(page, {config, name: 'open google'});
```

**Reply by ipadorusa** [reply]
thank you~

**Reply by BhaskarPhukan** [reply]
I need to add a few more things to config. how to do that ?
example :  const config = {
    extends: 'lighthouse:default',
    plugins: ['lighthouse-plugin-soft-navigation'],
  }
